swagger.yaml

```diff
@@ -124,6 +134,162 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/500Response"
+  /users:
+    post:
+      tags:
+        - User
+      summary: Register user into system
+      parameters:
+        - name: createUser
+          in: body
+          description: Create user
+          schema:
+            $ref: "#/definitions/userCreated"
```

Comment:
Why the name is past sentence `userCreated`, This is request body not the response? Please refer to POST /tutorials API and follow same best practices.